config/config.json

```diff
@@ -75,7 +75,7 @@
           "apiBasePath": "https://api.stage.openshift.com"
         },
         "ads": {
-          "editorsBasePath": "/beta/apps/ads-editors-build"
+          "editorsBasePath": "http://localhost:9011"
```

Comment:
FYI this is the endpoint used for local development.  This will assume that the developer is running the ads-editors angular application locally on port 9011 (the default port for that project).

Note: the upstream project is `apicurio-studio-editors` and the midstream is `ads-editors`.